scripts/cluster-helpers.sh

```diff
@@ -277,3 +280,20 @@ function check_prereq() {
 		fi
 	fi
 }
+
+# create kubernetes secret
+function create_secret() {
+
+	namespace="$1"
+	# create a kube secret each time app is deployed
+	kubectl create secret docker-registry hpodockersecret --docker-username="${REGISTRY_USERNAME}" \
+	--docker-server="${REGISTRY}" --docker-email="${REGISTRY_EMAIL}"  --docker-password="${REGISTRY_PASSWORD}" \
+	-n ${namespace}
+
+	# link the secret to the service account
+	if [ "${cluster_type}" == "minikube" ]; then
+		kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "hpodockersecret"}]}' -n ${namespace}
+	else
+		oc secrets link default hpodockersecret --for=pull -n ${namespace}
```

Comment:
As I mentioned previously, we cannot use `oc` in common code. This needs to be kubectl

Reply:
Updated now